app/views/docs/getting-started-for-server.phtml

```diff
@@ -318,7 +319,6 @@ $client
         <h3>Python</h3>
         <div class="ide margin-top-small" data-lang="python" data-lang-label="Python SDK">
             <pre class="line-numbers"><code class="prism language-python" data-prism>from appwrite.client import Client
-from appwrite.services.users import Users
```

Comment:
Was this deletion on purpose?

Reply:
yes, this is SDK init section, no need to import users here

Reply:
Okay